libr/core/cmd_search.inc.c

```diff
@@ -812,7 +812,7 @@ R_API RList *r_core_get_boundaries_prot(RCore *core, R_UNUSED int perm, const ch
 	snprintf (bound_from, sizeof (bound_from), "%s.%s", prefix, "from");
 	snprintf (bound_to, sizeof (bound_to), "%s.%s", prefix, "to");
 	const ut64 search_from = r_config_get_i (core->config, bound_from),
-	      search_to = r_config_get_i (core->config, bound_to);
+		  search_to = r_config_get_i (core->config, bound_to);
```

Comment:
split in two statements, dont do two assignemnts in one line

Reply:
indent ONLY with tabs.

Reply:
Probably a leftover from someone's commit, fixed by https://github.com/radareorg/radare2/pull/23359/commits/a32873816503db175c5ef9284f9b15d4e2263331